search.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:flutter/material.dart';
+// i am not sure what this functionality filters/ searches for?? videos? users? livestream?
+class Video {
+  final String title;
+  final String url;
+  final String category;
+
+  const Video({
+    required this.title,
+    required this.url,
+    required this.category,
+  });
+}
+//I am unsure about what needs to happen here: Import the data maybe?
+const List<Video> allVideos = [
+  Video(
+    title: 'Breathing exercise ',
+    url: 'https://thc.com/breathing.mp4',
+    category: 'Breathing',
+  ),
+  Video(
+    title: 'Video 2',
+    url: 'https:..mp4',
+    category: 'Meditation',
+  ),
+  // Do we have a databases for video's /how will this work?
+];
+
+class MainPage extends StatefulWidget {
+  @override
+  _MainPageState createState() => _MainPageState();
```

Comment:
Dart has a [linter rule](https://dart.dev/tools/linter-rules/library_private_types_in_public_api) that basically says "public functions should have public return types".

Luckily, it's a really quick fix:

```suggestion
  State<MainPage> createState() => _MainPageState();
```